wrapper/CSharp/wolfSSL_CSharp/wolfSSL.cs

```diff
@@ -417,6 +459,8 @@ public void free()
 
         public static readonly int SUCCESS = 1;
         public static readonly int FAILURE = 0;
+        public static readonly int WOLFSSL_SNI_HOST_NAME = 0;
+        public static readonly int WOLFSSL_SNI_HOST_NAME_OUTER = 0;

```

Comment:
Can you please remove `WOLFSSL_SNI_HOST_NAME_OUTER` here and also in ssl.h. I cannot find any place it is used and its a duplicate value in the enum, which is odd.